hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapLoaderTest.java

```diff
@@ -72,6 +72,7 @@ public void testSenderAndBackupTerminates_AfterInitialLoad() throws InterruptedE
         HazelcastInstance[] ownerAndReplicas = findOwnerAndReplicas(instances, name);
         ownerAndReplicas[0].getLifecycleService().terminate();
         ownerAndReplicas[1].getLifecycleService().terminate();
+        waitUntilActive(ownerAndReplicas[3], 15);
```

Comment:
yes, awaiting cluster size make sense. to ensure terminations of others are observed by alive one. 
there's already an assertion for that: `assertClusterSizeEventually()`